src/Tables/Templates/dataImports.json

```diff
@@ -12,6 +12,11 @@
             "event": "download-rejected",
             "class": "is-row-button has-text-danger"
         },
+        {
+            "type": "global",
```

Comment:
this should be per line, with a slot and disabled where it's not the case (`cancel`)